### ISSUE TITLE
How do you change Adam learning rate since the latest commits ?

### ISSUE THREAD

**Issue by divideconcept** [pytorch/pytorch#35640]
## 🐛 Bug

Since this commit : https://github.com/pytorch/pytorch/commit/76035f050b215d0606fe786901dcd07b5c9544fe#diff-b8c53e7a2010d3dae3200c9911950551
It's no longer possible to directly access the options variable of Adam, making impossible to change the learning rate on the fly with libtorch.
How can I change the learning rate ?

## To Reproduce

1. Install the latest nightly of libtorch
2. Create an Adam optimizer
3. Try to access its .options variable to change the learning rate : options does not exist anymore

**Comment by divideconcept** [comment]
@anjali411

**Comment by emcastillo** [comment]
You can access the `param_groups` attribute of the optimizer and change it there.
`adam_optimizer.param_groups[0]['lr']`
AFAIK this is the recommended way.

**Comment by divideconcept** [comment]
Thanks for the hint; I was actually referring to libtorch (not PyTorch), but seems that the C++ equivalent is: static_cast<torch::optim::AdamOptions&>(adam_optimizer.param_groups()[0].options()).lr()

**Comment by ailzhang** [comment]
cc: the author of the commit @anjali411 to confirm whether this is expected or not. Thanks!
